Dockerfile

```diff
@@ -1,41 +1,45 @@
+ARG HOMEDIR=/opt/gatekeeper
+
 #
-# Builder image
+# Builder
 #
 
 FROM golang:1.14.4 AS build-env
```

Comment:
I would use current Golang - image `golang:1.15` - so builder uses the latest minor Golang version. Minor versions shouldn't have breaking changes, so it should be safe.

Reply:
i would like to update to 15 all things in repo in next release, there is enough changes for now i think in this